lib/features/starting_page/presentation/twake_welcome/twake_welcome_view.dart

```diff
@@ -0,0 +1,65 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:get/get.dart';
+import 'package:linagora_design_flutter/linagora_design_flutter.dart';
+import 'package:tip_dialog/tip_dialog.dart';
+import 'package:tmail_ui_user/features/starting_page/presentation/twake_welcome/twake_welcome_controller.dart';
+import 'package:tmail_ui_user/features/starting_page/presentation/twake_welcome/twake_welcome_view_style.dart';
+import 'package:tmail_ui_user/main/localizations/app_localizations.dart';
+import 'package:tmail_ui_user/main/utils/app_config.dart';
+
+class TwakeWelcomeView extends GetWidget<TwakeWelcomeController> {
+
+  const TwakeWelcomeView({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    SystemChrome.setPreferredOrientations([
+      DeviceOrientation.portraitUp,
+      DeviceOrientation.portraitDown
+    ]);
```

Comment:
If our app supports landscape mobile mode, make sure to reset this config when this screen closes.

Reply:
It must be so.